lib/libc/stdio/lib_sscanf.c

```diff
@@ -511,7 +527,9 @@ int vsscanf(FAR const char *buf, FAR const char *fmt, va_list ap)
 						width  = fwidth;
 					}
 
+#ifndef CONFIG_DYNAMIC_ALLOC_ENABLE
 					width = MIN(sizeof(tmp) - 1, width);
+#endif
```

Comment:
This is similar to the "%s" case. In this case we are using regex based operations to determine the length of the input characters to be scanned. In any case, the code in line 531 needs to be removed, as we are not copying data into the variable "tmp". Instead we copy data into the variable "tv".

Reply:
Thanks
filtered other changes except string handling case.